Sources/SmileID/Classes/DocumentVerification/Model/OrchestratedDocumentVerificationViewModel.swift

```diff
@@ -121,17 +121,38 @@ internal class IOrchestratedDocumentVerificationViewModel<T, U: JobResult>: Obse
 
         let zip: Data
         do {
-            let savedFiles = try LocalStorage.saveDocumentImages(
-                front: documentFrontFile,
-                back: documentBackFile,
+            var allFiles = [URL]()
+            let frontDocumentUrl = try LocalStorage.createDocumentFile(jobId: jobId, document: documentFrontFile)
+            allFiles.append(frontDocumentUrl)
+            var backDocumentUrl: URL?
+            if let documentBackFile = documentBackFile {
+                let url = try LocalStorage.createDocumentFile(jobId: jobId, document: documentBackFile)
+                backDocumentUrl = url
+                allFiles.append(url)
+            }
+            allFiles.append(selfieFile)
+            var livenessImagesUrl = [URL]()
```

Comment:
imo it's a tiny bit confusing to have this extra variable here. I think we can remove it